package.json

```diff
@@ -28,11 +28,17 @@
   },
   "homepage": "https://github.com/lob/hapi-bookshelf-total-count#readme",
   "devDependencies": {
```

Comment:
I meant these too:

https://github.com/lob/hapi-sanitize-payload/blob/master/package.json
https://github.com/lob/hapi-default-payload/blob/master/package.json

So Hapi should be both a peer dependency to declare which version of Hapi this plugin works with, and a dev dependency so it is installed with tests. Correct?